From 59342ecdef76ef37eb71614d5e6f082cb8cd568b Mon Sep 17 00:00:00 2001
From: Sebastian Panceac <sebastian@resin.io>
Date: Thu, 2 Aug 2018 13:32:58 +0200
Subject: [PATCH] Use modules_install as wanted by yocto

Upstream-Status: Pending
Signed-off-by: Sebastian Panceac <sebastian@resin.io>

---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index eedf217..7fb1a8a 100644
--- a/Makefile
+++ b/Makefile
@@ -49,6 +49,9 @@ endif
 all: compiletime.h
 	$(MAKE) -C $(KDIR) M=$(PWD)  modules
 
+modules_install:
+	$(MAKE) -C $(KDIR) M=$(PWD)  modules_install
+
 compiletime.h:
 	echo "#define COMPILETIME \""`date`"\"" > compiletime.h
 
